src/client/datascience/commands/activeEditorContext.ts

```diff
@@ -211,7 +232,7 @@ export class ActiveEditorContextService implements IExtensionSingleActivationSer
     }
     private updateMergedContexts() {
         this.interactiveOrNativeContext
-            .set(this.nativeContext.value === true && this.interactiveContext.value === true)
+            .set(this.nativeContext.value === true || this.interactiveContext.value === true)
```

Comment:
Don't believe this is actually being used, but just noticed this when browsing the file so I fixed it.